force-app/main/default/lwc/cartesianAxis/cartesianAxis.js

```diff
@@ -78,15 +78,17 @@ export default class CartesianAxis extends BaseAxis {
     return this._content.ticks.min;
   }
   set ticksMin(v) {
-    this._content.ticks.min = Number(v) || v;
+    const numberValue = Number(v);
+    this._content.ticks.min = numberValue === 0 ? 0 : numberValue || v;
```

Comment:
Actually the solution should be just to do this:
```suggestion
    this._content.ticks.min = Number(v);
```
Because the ticks.min attribute only accepts numbers, we don't need the default value as a fallback.